js/src/collections/map/indicators_by_target.js

```diff
@@ -0,0 +1,22 @@
+var Handlebars = require('handlebars');
+var _ = require('lodash');
+var CartoDBCollection = require('../../lib/cartodb_collection.js');
+var CONFIG = require('../../../config.json');
+
+var SQL = Handlebars.compile(require('../../queries/map/indicators_by_target.hbs'));
+
+var IndicatorConfigs = CartoDBCollection.extend({
+  user_name: CONFIG.cartodb.user_name,
+
+  url: function() {
+    var query = SQL();
+    return this._urlForQuery(query);
+  },
+
+  parse: function(data) {
```

Comment:
If the parse function is this, (default one), you don't need to declare it because it's already declare it in CDB Collection. You only should change this if the parse is different from the inherit one.